static/js/click_handlers.js

```diff
@@ -423,12 +423,6 @@ exports.initialize = function () {
         $(".tooltip").remove();
     });
 
-    $("#subscriptions_table").on("click", ".exit, #subscription_overlay", function (e) {
-        if ($(e.target).is(".exit, .exit-sign, #subscription_overlay, #subscription_overlay > .flex")) {
-            subs.close();
-        }
-    });
-
```

Comment:
Should we also inline `exports.close()`  here into the `overlays.open_overlay` call?  I think we now only have one caller of that function...